### DISCUSSION TITLE
Project logo

### DISCUSSION THREAD

**Discussion by sadortun** [GoPlan-Finance/goplan-app#18] [Show and tell] [answered]
From @sdaniels1288  #12 

![image](https://user-images.githubusercontent.com/3279106/110056215-61562280-7d2c-11eb-9137-fda1897f3556.png)

![image](https://user-images.githubusercontent.com/3279106/110056222-66b36d00-7d2c-11eb-8dd2-d221e2583fda.png)

**Comment by sadortun** [comment]
@sdaniels1288 

> but initial thoughts?
ITS BIG ! 😆 

I would probably not include the word `finance` in the logo itself

(also trying to see if GitHub discussions are useful :)

**Reply by sdaniels1288** [reply]
As far as size goes, it's originally SVG so it can scale as needed. Mainly looking for input on typography, elements, etc.

**Reply by sadortun** [reply]
I like the 2nd one, you could change the arrow color for green/red/green

**Comment by mkitzmann** [comment, marked as answer]
I had already created a simple logo for the mockup that we could use as well:

![logo (2)](https://user-images.githubusercontent.com/35574021/110075856-c8eb8c80-7d83-11eb-986f-56a72a1d033b.png)

**Reply by sdaniels1288** [reply]
I didn't realize that was a logo per se. I like it though. Is it something we could extract as an SVG or something so that we could use it outside of the app/site for other things?

**Reply by sdaniels1288** [reply]
Never mind. Answered my own question. I'm all in favor of using this.

**Reply by sadortun** [reply]
I like it, and it's good enough for now. It's already the Organization logo and the Discord server image :)

**Reply by sdaniels1288** [reply]
Lol, I missed it. It's good though. Nice job.

**Comment by mkitzmann** [comment]
SVG with text and icon only
[portfolio tracking logo.zip](https://github.com/GoPlan-Finance/GoPlan-app/files/6093067/portfolio.tracking.logo.zip)
